### PR TITLE
disable ligature in lua snippets

### LINKED ISSUE #253: Disable ligatures in the CSS
In the CSS, you use the Fira Sans font, which has programming ligatures. However those ligatures don't make sense with all languages (for instance the `~=` ligature is terrible in Lua, since `~=` means different but the ligature is the "almost equal" symbol).

I suggest you add `font-variant-ligatures: none;` to the `.CodeMirror` block of the CSS to fix this.

### PR DESCRIPTION
**Pull request by rodmoioliveira**
Resolve #253

1. run  the project: `clj -A:figwheel --build dev --repl`
2. go to `http://localhost:5014/lua-dbg.html`

### CODE REVIEW

File: resources/public/lua-dbg.html [hunk 1]
```diff
@@ -1,34 +1,46 @@
...
-    <title>KLIPSE: a simple and elegant online cljs compiler and evaluator</title>
-    <link rel='shortcut icon' type='image/x-icon' href='img/klipse.png' />
-    <link rel="stylesheet" type="text/css" href="css/codemirror.css">
+    <meta charset="UTF-8" />
+    <title>
+      KLIPSE: a simple and elegant online cljs compiler and evaluator
+    </title>
+    <link rel="shortcut icon" type="image/x-icon" href="img/klipse.png" />
+    <link rel="stylesheet" type="text/css" href="css/codemirror.css" />
+    <link rel="stylesheet" type="text/css" href="css/lua.css" />
```
Comment (rodmoioliveira) [inline resources/public/lua-dbg.html:10]: Captura de tela em 2020-08-08 00-33-42

**Comment by viebel** [comment]
Please update the part of readme about Lua https://github.com/viebel/klipse/blob/master/README.md#lua with detailed explanations

**Comment by rodmoioliveira** [comment]
> Please update the part of readme about Lua https://github.com/viebel/klipse/blob/master/README.md#lua with detailed explanations

@viebel, I have added the link tag to the lua.css that disable ligatures in Lua snippets. See if that is enough. Regards.

File: README.md [hunk 2]
```diff
@@ -200,10 +200,11 @@ Here is an example of a [tiny reagent demo project](https://gitlab.com/viebel/cl
 
 ```html
 <link rel="stylesheet" type="text/css" href="https://storage.googleapis.com/app.klipse.tech/css/codemirror.css">
+<link rel="stylesheet" type="text/css" href="https://storage.googleapis.com/app.klipse.tech/css/lua.css">
```
Comment (rodmoioliveira) [inline README.md:203]: Link tag to disable ligatures in lua

**Pull request merged**